ublox/src/ubx_packets/packets.rs

```diff
@@ -1981,11 +1982,25 @@ struct EsfMeas {
 }
 
 #[ubx_packet_recv]
-#[ubx(class = 0x10, id = 0x03, fixed_payload_len = 16)]
-struct EsfRaw {
-    msss: u32,
+#[ubx(class = 0x10, id = 0x15, fixed_payload_len = 36)]
```

Comment:
These are fixed payload length so should be ok
https://content.u-blox.com/sites/default/files/products/documents/u-blox8-M8_ReceiverDescrProtSpec_UBX-13003221.pdf
page 295